src/ng/compile.js

```diff
@@ -1386,6 +1386,75 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return TTL;
   };
 
+  var COMMENT_DIRECTIVES_ENABLED = true;
```

Comment:
I agree, I just copied the TTL example and I'm not sure if there is any naming convention about it. I renamed it to `commentDirectivesEnabledConfig`.